sklearn/_build_utils/deprecated_modules.py

```diff
@@ -24,6 +24,12 @@
     ('_mean_shift', 'sklearn.cluster.mean_shift_', 'sklearn.cluster'),
     ('_optics', 'sklearn.cluster.optics_', 'sklearn.cluster'),
     ('_spectral', 'sklearn.cluster.spectral', 'sklearn.cluster'),
+
+    ('_base', 'sklearn.mixture.base', 'sklearn.mixture'),
```

Comment:
do we need this?

Reply:
` sklearn/mixture/base.py` was renamed into ` sklearn/mixture/_base.py`, so yes.

If a user tries to `from sklearn.mixture.base import something` it will get a warning saying that it should instead import stuff from `sklearn.mixture`, and if that doesn't work that means that what they were importing is now private and they should not use it.